src/main/java/com/sfuentes/Theater.java

```diff
@@ -0,0 +1,44 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Theater {
+
+  ArrayList<Integer> ages;
+  final double price = 20d;
+
+  public Theater(ArrayList<Integer> ages) {
```

Comment:
Please use the interface List instead of the implementation ArrayList

Reply:
fix

Reply:
fix